pyproject.toml

```diff
@@ -110,3 +110,42 @@ minversion = "6.0"
 testpaths = "iris_grib"
 xfail_strict = "True"
 
+[tool.ruff]
+# Exclude the following, in addition to the standard set of exclusions.
+# https://docs.astral.sh/ruff/settings/#exclude
+line-length = 88
+src = [
+    "iris-grib",
+    "docs",
+]
+
+[tool.ruff.format]
+preview = false
+
+[tool.ruff.lint]
+ignore = [
+    # NOTE: Non-permanent exclusions should be added to the ".ruff.toml" file.
+
+    # flake8-commas (COM)
+    # https://docs.astral.sh/ruff/rules/#flake8-commas-com
+    "COM812",  # Trailing comma missing.
+    "COM819",  # Trailing comma prohibited.
+
+    # flake8-implicit-str-concat (ISC)
+    # https://docs.astral.sh/ruff/rules/single-line-implicit-string-concatenation/
+    # NOTE: This rule may cause conflicts when used with "ruff format".
+    "ISC001",  # Implicitly concatenate string literals on one line.
+    ]
+preview = false
+select = [
+    "ALL",
+    # list specific rules to include that is skipped using numpy convention.
```

Comment:
@HGWright Perhaps replace with with a blank line separating it from line +141, then replace with:

```
    # pydocstyle (D)
    # https://docs.astral.sh/ruff/rules/multi-line-summary-first-line/
```